src/native/corehost/hostmisc/pal.unix.cpp

```diff
@@ -767,6 +767,7 @@ pal::string_t normalize_linux_rid(pal::string_t rid)
     pal::string_t rhelPrefix(_X("rhel."));
     pal::string_t alpinePrefix(_X("alpine."));
     pal::string_t rockyPrefix(_X("rocky."));
+    pal::string_t olPrefix(_X("ol."));
```

Comment:
@richlander @elinor-fung now that we've frozen the RID graph, we shouldn't need these changes, right?

Reply:
Correct.  Host should not care about distro-specific RIDs. This implementation is for legacy .NET versions.